plugins/lookup/etcd3.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2020, SCC France, Eric Belhomme <ebelhomme@fr.scc.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    author:
+    - Eric Belhomme <ebelhomme@fr.scc.com>
+
+    lookup: etcd3
+
+    version_added: "2.9.6"
+
+    short_description: get info from an etcd3 server
+
+    description:
+    - Retrieves data from an etcd3 server
+
+    options:
+        _terms:
+            description:
+            - the list of keys to lookup on the etcd3 server
```

Comment:
```suggestion
            - The list of keys to look up on the etcd3 server.
```
(similar adjustments for the other descriptions)